src/accelerate/components/List.tsx

```diff
@@ -97,16 +131,39 @@ export default function List( props: Props ) {
 							} }
 						/>
 					</div>
+					<div className="table-add-new">
+						<Dropdown
+							className=""
+							contentClassName=""
+							position="bottom right"
+							renderToggle={ ( { isOpen, onToggle } ) => (
+								<Button
+									variant="primary"
+									onClick={ onToggle }
+									aria-expanded={ isOpen }
+									className='dashicons-before dashicons-plus'
+								>
+									{ __( 'Add New', 'altis' ) }
+								</Button>
+							) }
+							renderContent={ ( { onClose } ) => (
+								<MenuGroup>
+									{/* <MenuItem onClick={ onAddNew( 'block' ) }>
+										{ __( 'Block', 'altis' ) }
+									</MenuItem> */}
```

Comment:
Why is this commented out? Might be good to expose if it helps drive the kind of activity we want to see?

Reply:
I don't think you can actually create a _reusable block_ right now ?

Reply:
The post type is `wp_block`, maybe that's why